src/app/components/food-counter/food-counter.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import { TreeNode } from '@circlon/angular-tree-component';
+import { TreeService } from 'src/app/services/tree.service';
+
+type PartialTreeNode = Partial<TreeNode> & {
+  name?: string;
+};
+
+@Component({
+  selector: 'app-food-counter',
+  templateUrl: './food-counter.component.html',
+  styleUrls: ['./food-counter.component.scss'],
+})
+export class FoodCounterComponent implements OnInit {
+  count = 0;
+
+  constructor(private treeSvc: TreeService) {
+    this._recursivelyCountNodes(
+      (this.treeSvc.getNodes() as unknown) as PartialTreeNode[]
```

Comment:
When production gets merged into this branch the type returned from `getNodes()` is an `Observable`, so I don't think this will work.

Reply:
Okay, I'll try that before I merge it.